mesheryctl/internal/cli/root/config/config.go

```diff
@@ -74,22 +77,22 @@ func (mc *MesheryCtlConfig) CheckIfCurrentContextIsValid() (Context, error) {
 }
 
 // CheckIfCurrentVersionIsValid checks if current version is valid, return context as a string
-func CheckIfCurrentVersionIsValid(version string) (string, error) {
-
+func (mc *MesheryCtlConfig) CheckIfCurrentVersionIsValid() error {
```

Comment:
```suggestion
func (ctx *Context) ValidateVersion() error {
```

@metonymic-smokey I think you misread my previous suggestion, I meant `Context` struct not `MesheryCtlConfig` struct

Reply:
right, totally my bad. So sorry!